### PR TITLE
docs - ALTER/CREATE/DROP VIEW sql ref page updates

### PR DESCRIPTION
**Pull request by lisakowen**
bring ALTER/CREATE/DROP VIEW sql ref pages up to postgres 12 content. there looks to be some greenplum-specific content on the CREATE page.

NOTE: these updates assume that greenplum supports updatable views.

CREATE VIEW:
- "WITH [ CASCADED | LOCAL ] CHECK OPTION" is also in 9.4, but we didn't document it. i added the description from pg12 page, but omitted a note about INSTEAD OF triggers because i saw a comment in the code that greenplum can't support this.
- the Notes include a few statements that are not in the postgres docs,  search for XXX.  are these valid for greenplum 7?

doc review site link (behind vpn), can navigate to others from here:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-view/greenplum-database/GUID-ref_guide-sql_commands-ALTER_VIEW.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/sql_commands/CREATE_VIEW.html.md [hunk 1]
```diff
@@ -49,58 +46,135 @@ name
...
 
-However, functions called in the view are treated the same as if they had been called directly from the query using the view. Therefore the user of a view must have permissions to call any functions used by the view.
+Functions called in the view are treated the same as if they had been called directly from the query using the view. Therefore the user of a view must have permissions to call any functions used by the view.
 
-If you create a view with an `ORDER BY` clause, the `ORDER BY` clause is ignored when you do a `SELECT` from the view.
+XXX If you create a view with an `ORDER BY` clause, the `ORDER BY` clause is ignored when you `SELECT` from the view.
```
Comment (QingMa-gp) [inline gpdb-doc/markdown/ref_guide/sql_commands/CREATE_VIEW.html.md:92]: The "order by" clause of the view will be ignored only when there is an "order by" clause in the "select from view" statement.
```
postgres=# create table test_t (a int, b int);
CREATE TABLE
postgres=# create view test_v as select * from test_t order by b;
CREATE VIEW
postgres=# explain select * from test_v;
                                  QUERY PLAN                                  
------------------------------------------------------------------------------
 Gather Motion 3:1  (slice1; segments: 3)  (cost=0.00..431.00 rows=1 width=8)
   Merge Key: b
   ->  Sort  (cost=0.00..431.00 rows=1 width=8)
         Sort Key: b
         ->  Seq Scan on test_t  (cost=0.00..431.00 rows=1 width=8)
 Optimizer: Pivotal Optimizer (GPORCA)
(6 rows)

postgres=# explain select * from test_v order by a;
                                  QUERY PLAN                                  
------------------------------------------------------------------------------
 Gather Motion 3:1  (slice1; segments: 3)  (cost=0.00..431.00 rows=1 width=8)
   Merge Key: a
   ->  Sort  (cost=0.00..431.00 rows=1 width=8)
         Sort Key: a
         ->  Seq Scan on test_t  (cost=0.00..431.00 rows=1 width=8)
 Optimizer: Pivotal Optimizer (GPORCA)
(6 rows)

```

Comment (QingMa-gp) [reply gpdb-doc/markdown/ref_guide/sql_commands/CREATE_VIEW.html.md]: Sorry, my previous description may not have been accurate. When both `select ... from view` and `create view` have an `order by` clause, the `order by` clause in the `create view` statement will be ignored. Based on the above case, the second query ignores the `order by b` clause.

Comment (lisakowen) [reply gpdb-doc/markdown/ref_guide/sql_commands/CREATE_VIEW.html.md]: thank you for clarifying, @QingMa-gp

**Review by QingMa-gp — APPROVED** [review]
LGTM.

**Review by dyozie — APPROVED** [review]

**Pull request merged**